packages/debug/src/browser/console/debug-console-items.tsx

```diff
@@ -103,6 +103,10 @@ export class ExpressionContainer implements CompositeConsoleItem {
             const response = await this.session!.sendRequest('variables', { variablesReference, filter, start, count });
             const { variables } = response.body;
             const names = new Set<string>();
+            for (const item of result) {
+                const variable = item as DebugVariable
+                names.add(variable.name)
```

Comment:
These two statements are missing their semicolons. Please activate the ESLint extension in the Theia repo to catch these issues beforehand.